### DISCUSSION TITLE
read after write consistency in s3

### DISCUSSION THREAD

**Discussion by chenqin** [varchar-io/nebula#58] [General]
There are numerous issue of not able to read/list after write on s3 (aka eventual consistency) 
Notice in recent s3 were able to provide such strong grantee.

https://aws.amazon.com/blogs/aws/amazon-s3-update-strong-read-after-write-consistency/

How can nebula benefit from such improvement and possible reduce complexity of metadata store.

@shawncao

**Comment by shawncao** [comment]
wow, this is big move for S3, they are filling the gap where GCS runs ahead, :)
